contracts/SinglePlayerCommit.sol

```diff
@@ -61,19 +60,10 @@ contract SinglePlayerCommit is ChainlinkClient, Ownable {
     event CommitmentEnded(address committer, bool met, uint256 amountPenalized);
     event Deposit(address committer, uint256 amount);
     event Withdrawal(address committer, uint256 amount);
-    event RequestActivityDistanceFulfilled(
-        bytes32 indexed requestId,
-        uint256 indexed distance
-    );
-    event ActivityUpdated(
-        string name, 
-        bytes32 activityKey, 
-        address oracle, 
-        bool allowed,
-        bool exists);
+    event RequestActivityDistanceFulfilled(bytes32 indexed requestId, uint256 indexed distance);
```

Comment:
Is this the event our app is waiting on? If so, should we include the committer's address as well?